src/Core/TileMesh.js

```diff
@@ -101,6 +103,43 @@ class TileMesh extends THREE.Mesh {
             this.material.updateLayersUniforms();
         }
     }
+
+    /**
+     * Refresh a material of this tile, associated with a specific layer. Only
+     * works with ColorLayer and ElevationLayer for now.
+     *
+     * @param {ColorLayer|ElevationLayer} layer - The layer associated to the
+     * material.
+     * @param {View} [view] - The view to update, don't forget to call
+     * `view.notifyChange` otherwise.
+     * @param {boolean} [traverse=false] - Refresh the children of the tile - if
+     * any.
+     */
+    refreshMaterial(layer, view, traverse = false) {
+        this.isRefreshing = true;
+
+        if (layer.isColorLayer || layer.isElevationLayer) {
+            const materialLayer = this.material.getLayer(layer.id);
+            if (materialLayer) {
+                materialLayer.textures.forEach(t => Cache.delete(layer.source.uid, layer.id, t.extent.toString('-')));
+            }
+
+            this.material.removeLayer(layer.id);
+            this.layerUpdateState[layer.id].forceUpdate();
+
+            if (view) {
+                view.notifyChange(this);
```

Comment:
I don't think this is the best place to refresh the view.

Reply:
Moved to a better place